radio/src/translations/ua.h

```diff
@@ -0,0 +1,1257 @@
+#define TR_RADIO_DATA_UNRECOVERABLE    TR3("Невірні установки радіо","Недійсні установки радіо", "Не можу проситати правильні установки радіо")
+#define TR_EEPROMFORMATTING            "Форматую EEPROM"
+#define TR_STORAGE_FORMAT              "Підготовка сховища"
+#define TR_EEPROMOVERFLOW              "Переповнення EEPROM"
+#define TR_RADIO_SETUP                 "УСТАНОВКИ РАДІО"
+#define TR_MENUTRAINER                 "ТРЕЙНЕР"
+#define TR_MENUSPECIALFUNCS            "ГЛОБАЛЬНІ ФУНКЦІЇ"	
+#define TR_MENUVERSION                 "ВЕРСІЯ"
+#define TR_MENU_RADIO_SWITCHES         TR("ПЕРЕМИКАЧІ", "ТЕСТ ПЕРЕМИКАЧІВ")
+#define TR_MENU_RADIO_ANALOGS          TR("АНАЛОГ", "ТЕСТ АНАЛОГУ")
+#define TR_MENU_RADIO_ANALOGS_CALIB    "АНАЛОГИ ВІДКАЛІБРОВАНІ"
+#define TR_MENU_RADIO_ANALOGS_RAWLOWFPS "СИРІ АНАЛОГИ (5 Hz)"
+#define TR_MENUCALIBRATION             "КАЛІБРУВАННЯ"
+#define TR_MENU_FSWITCH                "КОРИСТУВАЦЬКІ ПЕРЕМИКАЧІ"
+#if defined(COLORLCD)
+  #define TR_TRIMS2OFFSETS             "Трим => Субтрим"
+#else
+  #define TR_TRIMS2OFFSETS             "\006Трим => Трим"
+#endif
+#define TR_CHANNELS2FAILSAFE           "Канали=>АварРежим"
+#define TR_CHANNEL2FAILSAFE            "Канал=>АварРежим"
+#define TR_MENUMODELSEL                TR("ВИБІРМОД", "ВИБІР МОДЕЛІ")
+#define TR_MENU_MODEL_SETUP            TR("НАЛАШТУВ", "НАЛАШТУВАННЯ МОДЕЛІ")
+#if defined(SURFACE_RADIO)
+#define TR_MENUFLIGHTMODES             "РЕЖИМИ ВОДІННЯ"
+#define TR_MENUFLIGHTMODE              "РЕЖИМ ВОДІННЯ"
+#else
+#define TR_MENUFLIGHTMODES             "РЕЖИМИ ПОЛЬОТУ"
+#define TR_MENUFLIGHTMODE              "РЕЖИМ ПОЛЬОТУ"
+#endif
+#define TR_MENUHELISETUP               "РЕЖИМ ГЕЛІКОПТЕРА"
+
+  #define TR_MENUINPUTS                "ВХОДИ"
+  #define TR_MENULIMITS                "ВИХОДИ"
+#define TR_MENUCURVES                  "КРИВІ"
+#define TR_MENUCURVE                   "КРИВА"
+#define TR_MENULOGICALSWITCH           "ЛОГІЧНІ ПЕРЕМИКАЧ"
+#define TR_MENULOGICALSWITCHES         "ЛОГІЧНІ ПЕРЕМИКАЧІ"
+#define TR_MENUCUSTOMFUNC              "СПЕЦІАЛЬНІ ФУНКЦІЇ"
+#define TR_MENUCUSTOMSCRIPTS           "КОРИСТУВАЦЬКИ СКРИПТИ"
+#define TR_MENUTELEMETRY               "ТЕЛЕНМЕТРІЯ"
+#define TR_MENUSTAT                    "СТАТИСТИКА"
+#define TR_MENUDEBUG                   "ДЕБАГ"
+#define TR_MONITOR_CHANNELS1           "МОНИТОР КАН 1-8"
+#define TR_MONITOR_CHANNELS2           "МОНИТОР КАН 9-16"
+#define TR_MONITOR_CHANNELS3           "МОНИТОР КАН 17-24"
+#define TR_MONITOR_CHANNELS4           "МОНИТОР КАН 25-32"
+#define TR_MONITOR_SWITCHES            "МОНИТОР ЛОГІЧНИХ ПЕРЕМИКАЧІВ"
+#define TR_MONITOR_OUTPUT_DESC         "Виходи"
+#define TR_MONITOR_MIXER_DESC          "Мікшери"
+  #define TR_RECEIVER_NUM              TR("RxNum", "No. приймача")
+  #define TR_RECEIVER                  "Приймач"
+#define TR_MULTI_RFTUNE                TR("Налаштув.частоти", "Точне налаштування Радіочастоти")
+#define TR_MULTI_RFPOWER               "RF power"		/* use english */
```

Comment:
The word `power` here also could be translated in my opinion
```suggestion
#define TR_MULTI_RFPOWER               "RF потужність"		/* use english */
```